app/src/main/java/ch/epfl/qedit/model/User.java

```diff
@@ -86,17 +94,43 @@ public String getFullName() {
         return firstName + " " + lastName;
     }
 
-    public Role getRole() {
-        return role;
+    public int getScore() {
+        return score;
+    }
+
+    public int getSuccess() {
+        return success;
+    }
+
+    public int getAttempt() {
+        return attempt;
+    }
+
+    public void incrementScore(int points) {
+        if (points < 0)
```

Comment:
Maybe we could have penality points one day

Reply:
Yes okey, i will make the condition (score - points) >= 0

Reply:
(In this case the condition should be `score + points >= 0`)

Reply:
yes yes it s just that to keep nice method names i have added a decrementScore and here the condition is score - point